README.md

```diff
@@ -26,8 +26,42 @@ In order to run this task, couple elements are required within the infrastructur
   * `terraform-apply`: Terraform job similar to the plan one, but will actually create/update everything that needs to. Please see the plan diff for a better understanding.
   * `terraform-destroy`: :warning: Terraform job meant to destroy the whole stack - **NO CONFIRMATION ASKED**. If triggered, the full project **WILL** be destroyed. Use with caution.
 
+**Variations**
+
+There is 4 versions of the pipeline supported in this stack:
+  * AWS: `pipeline.yml` + `variables.sample.yml` (default)
+  * S3-like compatible backends: `pipeline-s3-compatible.yml` + `variables-s3-compatible.sample.yml`
+  * GCP: `pipeline-gcp.yml` + `variables-gcp.sample.yml`
+  * Azure: `pipeline-azure.yml` + `variables-azure.sample.yml`
+
+Currently, Cycloid UI doesn't support switching between different pipeline variations, you will have to manually copy+paste pipeline+variables from GIT inside the Cycloid UI to use them.
```

Comment:
Maybe we should talking about `Cycloid console` instead of `Cycloid UI` ?

Reply:
yeah sure